src/infra/auth/jwt.ts

```diff
@@ -103,21 +119,23 @@ export const loginUserXHR = async (
 	extraParams?: ExtraParams,
 ): Promise<void> => {
 	const token = await createSessionToken(userId, extraParams);
-	sendXHRToken(res, token, statusCode);
+	sendXHRToken(res, token, statusCode, extraParams?.tx);
 };
 
 export const updateUserXHR = async (
 	res: Response,
 	req: Request,
+	{ tx }: { tx?: Tx } = {},
 ): Promise<void> => {
 	await getUser(req, false);
```

Comment:
It might not make any difference based on how we use it atm afaict, but it makes sense to be passing it here as well.
At minimum it should avoid creating an extra read transaction.
:+1: